src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -0,0 +1,377 @@
+                Help Scout
+              </StyledSafeLink>
+              . If you opt-in, we will load their program onto Aragon.
+              <br />
+              Help Scout is a{' '}
+              <StyledSafeLink href="https://bcorporation.net/directory/help-scout">
+                Public Benefit Corp
+              </StyledSafeLink>
+              .
+            </p>
+            <Button
+              mode="strong"
+              wide
+              onClick={onOptIn}
+              css={`
+                margin-top: ${1 * GU}px;
+                font-size: 15px;
+              `}
+            >
+              Yes, I'd like help
+            </Button>
+          </Body>
+        </main>
+      </Wrapper>
+    </animated.div>
+  )
+})
+
+OptInDialogue.propTypes = {
+  onClose: PropTypes.func.isRequired,
+  onOptIn: PropTypes.func.isRequired,
+}
+
+const Wrapper = styled.aside`
+  background: #fff;
+  border: 1px solid rgba(209, 209, 209, 0.5);
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.15);
+  border-radius: 3px;
+  width: calc(100vw - ${4 * GU}px);
+  position: absolute;
+  bottom: 100%;
+  /* zeroY is relative to beacon container full viewport height
+  * minus 4 GUs (64px) top plus the relative position of zeroY to the
+  * edge of the viewport (60px) plus one GU for margin (16px) */
+  top: calc(-100vh + 140px);
+  /* zeroX is relative to beacon container full viewport width
+  * minus 2 GUs on each side (32px) plus the position of zeroX
+  * relative to the edge of the viewport (60px) */
+  left: calc(-100vw + 92px);
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  height: calc(100vh - 60px - ${2 * GU}px - ${4 * GU}px - ${4 * GU}px);
+  overflow: hidden;
+
+  ${breakpoint(
+    'medium',
+    `
+      width: 350px;
+      height: 600px;
+      position: unset;
+    `
+  )}
+`
+
+const Header = styled.header`
+  border-top-right-radius: 2px;
+  border-top-left-radius: 2px;
+  height: 150px;
+  min-height: 150px;
+  background-color: transparent;
+  background-image: linear-gradient(
+    180deg,
+    ${theme.gradientStart},
+    ${theme.gradientEnd}
+  );
+  color: ${theme.gradientText};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`
+
+const Body = styled.div`
+  min-height: 350px;
+  line-height: 22px;
+  font-size: 15px;
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: space-around;
+`
+
+const StyledSafeLink = styled(SafeLink).attrs({ target: '_blank' })`
+  text-decoration: none;
+  color: ${theme.accent};
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+`
+
+const RoundButton = styled(Button).attrs({ mode: 'strong' })`
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.15);
+  border-radius: 50%;
+  width: 60px;
+  height: 60px;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const AnimatedDiv = styled(animated.div)`
```

Comment:
Maybe we could be a bit more specific than `AnimatedDiv`, what do you think about `RoundButtonIcon` or `RoundButtonIn`?